### ISSUE TITLE
Review All Developed Libraries Once TinyGo Completed Reflect.Value MapRange Development

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#9]
# Description
```
Please provide a short description of what feature you're looking forward to
enhance below. Please include the story behind your idea as well to give a
better visualization of your idea.
```

Appearently, TinyGo has yet to implement `(reflect.Value).MapRange()` as filed here https://github.com/tinygo-org/tinygo/issues/3104
While Go is not affected, we need to carefully proceed and review all codes once TinyGo facilitates this feature.

# Expected Behavior
```
Please specify the expected behavior of your requested enhancement. Some great
and helpful pointers are your expected interface (e.g. command patterns,
simple sketches of the user interface, etc).
```

MapRange is available.

# Current Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```

MapRange is denied by TinyGo due to absent of the feature.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Temporarily disabled TinyGo hestiaTESTING monteur recipe (e1e5b5cdb82fb349bd91410cee028e8ee5750a11)

**Comment by hollowaykeanho** [comment]
TinyGo hestiaTESTING monteur recipe reenabled after string slice switches was implemented (647e6fc8ae24b1e2b23a402068bc36b4e7cbce9c) and proven working properly.
